### PR TITLE
NVIDIA MPS documentation

### PR DESCRIPTION
**Pull request by mreso**
## Description

This PR adds a documentation for MPS with TorchServe

Fixes #(issue)
#2128

## Type of change

Please delete options that are not relevant.

- [X] Adds documentation

## Feature/Issue validation/testing

Please describe the Unit or Integration tests that you ran to verify your changes and relevant result summary. Provide instructions so it can be reproduced.
Please also list any relevant details for your test configuration.

- [ ] Test A
Logs for Test A

- [ ] Test B
Logs for Test B

## Checklist:

- [X] Did you have fun?
- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [X] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Review by msaroufim — APPROVED** [review]

**Comment by msaroufim** [comment]
Minor nit: Mind renaming the PR to NVIDIA MPS documentation? We don't know if torchserve works with Mac MPS yet lol

File: docs/mps.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+We use the following config.json for the benchmark, only overwriting the number of workers and the batch size accordingly.
+
+```
+{
+    "url":"/home/ubuntu/serve/examples/Huggingface_Transformers/model_store/BERTSeqClassification",
+    "requests": 10000,
+    "concurrency": 600,
+    "input": "/home/ubuntu/serve/examples/Huggingface_Transformers/Seq_classification_artifacts/sample_text_captum_input.txt",
+    "gpus": "1",
```
Comment (lxning) [inline docs/mps.md:47]: the [gpus](https://github.com/pytorch/serve/blob/master/benchmarks/benchmark-ab.py#L66) of the benchmark-ab.py is only applied on docker container.

Did all of the tests run on docker container?

Comment (mreso) [reply docs/mps.md]: @lxning As discussed, I added the remark regarding the single GPU instance and removed the parameter to avoid confusion.

**Review by HamidShojanazeri — APPROVED** [review]

File: docs/mps.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+![P3 benchmark, single worker](images/mps_p3_single.png)
+
+We can see that the throughput steady increases but for a batch size over eight we see diminishing returns.
+Finally, we deploy two workers on the P3 instance and compare running them with and without MPS.
+We can see that for batch size between 1 and 32 the throughput is consistently higher (up to +25%) for MPS enabled with the exception of batch size 16.
+
+![P3 benchmark, two workers](images/mps_p3_two_worker.png)
+
+## Summary
```
Comment (lxning) [inline docs/mps.md:86]: can we add some information like
once gpu utilization hits threshold (eg. 50%+), the throughput gain is limited.

**Pull request merged**